Homework11/WebTableTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System.Collections.ObjectModel;
+using System.Text;
+using Homework11.Configs;
+
+namespace Homework11
+{
+    public class WebTableTests
+    {
+
+        private IWebDriver _driver;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driver.Navigate().GoToUrl(Constants.webTablesPage);
+        }
+
+        [Test]
+        public void CheckValidation()
+        {
+            var addNewEntryButton = _driver.FindElement(By.XPath("//button[@id='addNewRecordButton']"));
+            addNewEntryButton.Click();
+            var submitButton = _driver.FindElement(By.XPath("//button[@id='submit']"));
+            submitButton.Click();
+            string validatedFormClass = "was-validated";
+            var formClass = _driver.FindElement(By.XPath("//form")).GetAttribute("class");
+            Assert.AreEqual(validatedFormClass, formClass);
+        }
+
+        [Test]
+        public void SearchEntry()
+        {
+            var searchInput = _driver.FindElement(By.XPath("//input[@id='searchBox']"));
+            FillInInput(searchInput, Constants.lastNameForSearch);
+            var expectedCell = _driver.FindElements(By.XPath("//div[contains(text(),'Vega')]"));
```

Comment:
I don't think it's a correct "check". It's 1 with "Vega" row before search. I would suggest compare results before and after searching (returned rows that contain "Vega")

Reply:
fixed